packages/flutter_driver/test/src/real_tests/flutter_driver_test.dart

```diff
@@ -591,6 +697,104 @@ void main() {
     });
   });
 
+  group('WebFlutterDriver with logCommunicationToFile', () {
+    late FakeFlutterWebConnection fakeConnection;
+    late WebFlutterDriver driver;
+    int driverId = 0;
+
+    setUp(() {
+      _cleanTestOutputFiles();
+      fakeConnection = FakeFlutterWebConnection();
+      fakeConnection.supportsTimelineAction = true;
+      fakeConnection.responses['waitFor'] = jsonEncode(makeFakeResponse(<String, dynamic>{'status': 'ok'}));
+    });
+
+    tearDown(() {
+      // Delete log file after test
+      final File file = File(p.join(testOutputsDirectory, 'flutter_driver_commands_$driverId.log'));
+      if (file.existsSync()) {
+        file.delete();
+      }
+      driverId++;
+    });
+
+    test('logCommunicationToFile = true', () async {
+      // Given
```

Comment:
Please remove these comments here and elsewhere

Reply:
@jonahwilliams I deleted these comments.